internal/clusterfeature/features/logging/common.go

```diff
@@ -50,6 +50,7 @@ const (
 
 	lokiOutputDefinitionName = "loki-output"
 	flowResourceName         = "banzai-logging-flow"
+	resourceLabelKey         = "service"
```

Comment:
I would recommend using a qualified key here, something like `banzaicloud.io/service`

Reply:
thx @pepov , i updated the key